src/components/Modal/Modal.cy.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Modal from "@/components/Modal/Modal";
+
+describe("General Modal Component", () => {
+    it("renders", () => {
+        cy.mount(
+            <Modal header="Modal Header" ariaLabel="Modal Label" isOpen={true} onClose={() => {}}>
```

Comment:
Is `ariaLabel` something that we should be implementing in `DataViewer.tsx`?

Reply:
We make it optional